src/release.ts

```
@@ -10,28 +60,46 @@ const defaultPrereleaseTag = 'rc';
 
 prompt.selectOption;
 
+export enum ReleaseType {
+  Release,
+  ReleaseCandidate,
+  BasePrerelease,
+}
+
+export async function tagBasePrerelease(prereleaseTag = defaultPrereleaseTag, releaseBranch = defaultReleaseBranch) {
```

just wondering if we should have the default prerelease tag  (RC) here. It might be better to just fail if not provided. Then in the buildsystem we can prompt for a PR number or other text

---

i do that in our buildsystem(i'll put up that PR as well) but putting RC as default for the base prerelease feels wrong as it will have the version number of a version that already exists. I'm considering calling it postreleases instead of base prereleases

---

```
export async function tagBasePrerelease() {
  const ans = await prompt.question('Enter PR number or prerelease name');
  if (!ans) {
    return false;
  }
  const isNumber = /^\d+$/.test(ans);
  return await release.tagBasePrerelease(isNumber ? `pr.${ans}` : ans);
}
```